build.sbt

```diff
@@ -7,7 +7,7 @@ homepage := Some(url("http://sangria-graphql.org"))
 licenses := Seq("Apache License, ASL Version 2.0" → url("http://www.apache.org/licenses/LICENSE-2.0"))
 
 scalaVersion := "2.12.6"
-crossScalaVersions := Seq("2.11.11", "2.12.6")
+crossScalaVersions := Seq("2.11.12", "2.12.6")
```

Comment:
I feel would would be better to avoid `2.11.12` for now. Recently I had some issues with it - it caused flaky builds.